app/assets/javascripts/multi-checkbox.es6

```diff
@@ -0,0 +1,53 @@
+/* global List */
+{
+  'use strict';
+
+  class MultiCheckbox {
+    constructor(el, config = {}) {
+      this.$el = el;
+      this.config = config;
+      this.itemCheckboxes = this.$el.find('[data-multi-checkbox="item"]');
+      this.allCheckbox = this.$el.find('[data-multi-checkbox="all"]');
+
+      this.bindEvents();
+    }
+
+    bindEvents() {
+      this.itemCheckboxes.on('change', this.handleItemChange.bind(this));
+      this.allCheckbox.on('change', this.handleAllChange.bind(this));
+    }
+
+    handleItemChange(event) {
+      const $el = $(event.currentTarget),
+      checked = this.getCheckedStatus($el);
```

Comment:
If we're going with the `,` approach for definitions, could we indent the lines that follow it? So

```es6
const $el = $(event.currentTarget),
  checked = this.getCheckedStatus($el);
```